src/zstd.mk

```diff
@@ -0,0 +1,27 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := zstd
+$(PKG)_WEBSITE  := https://github.com/facebook/$(PKG)
+$(PKG)_DESCR    := Zstandard is a fast lossless compression algorithm
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 1.3.2
+$(PKG)_CHECKSUM := ac5054a3c64e6510bc1ae890d05e3d271cc33ceebc9d06ac9f08105766d2798a
+$(PKG)_GH_CONF  := facebook/zstd,v
+$(PKG)_DEPS     := gcc
+
+define $(PKG)_BUILD
+    # build and install the library
+    cd '$(BUILD_DIR)' && $(TARGET)-cmake \
+        -DCMAKE_INSTALL_BINDIR=$(BUILD_DIR)/null \
+        -DCMAKE_INSTALL_MANDIR=$(BUILD_DIR)/null \
+        '$(SOURCE_DIR)/build/cmake'
+    $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)'
+    $(MAKE) -C '$(BUILD_DIR)' -j 1 install
+    $(if $(BUILD_SHARED), $(INSTALL) '$(BUILD_DIR)/lib/libzstd.dll' '$(PREFIX)/$(TARGET)/bin/')
+
+    # compile test
+    '$(TARGET)-gcc' \
+        -W -Wall -Werror -ansi -pedantic \
+        '$(TEST_FILE)' -o '$(PREFIX)/$(TARGET)/bin/test-$(PKG).exe' \
+        -L '$(PREFIX)/$(TARGET)/lib' -lzstd
```

Comment:
Install a *.pc file to avoid manual libs and make it easier for downstream packages:
```diff
+    # create pkg-config files
+    $(INSTALL) -d '$(PREFIX)/$(TARGET)/lib/pkgconfig'
+    (echo 'Name: $(PKG)'; \
+     echo 'Version: $($(PKG)_VERSION)'; \
+     echo 'Description: $($(PKG)_DESCR)'; \
+     echo 'Libs: -L$(PREFIX)/$(TARGET)/lib -l$(PKG)'; \
+     echo 'Cflags: -I$(PREFIX)/$(TARGET)/include';) \
+     > '$(PREFIX)/$(TARGET)/lib/pkgconfig/$(PKG).pc'

     # compile test
     '$(TARGET)-gcc' \
         -W -Wall -Werror -ansi -pedantic \
         '$(TEST_FILE)' -o '$(PREFIX)/$(TARGET)/bin/test-$(PKG).exe' \
-        -L '$(PREFIX)/$(TARGET)/lib' -lzstd
+        `'$(TARGET)-pkg-config' $(PKG) --cflags --libs`
```